ipynb/android-wloads/Android_YoutTube_Tests.ipynb

```diff
@@ -0,0 +1,500 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Populating the interactive namespace from numpy and matplotlib\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pylab inline\n",
+    "\n",
+    "import copy\n",
+    "import os\n",
+    "from time import sleep\n",
+    "\n",
+    "# Support to access the remote target\n",
+    "import devlib\n",
+    "from env import TestEnv\n",
+    "\n",
+    "from devlib.utils.android import adb_command\n",
+    "\n",
+    "# Support for trace events analysis\n",
+    "from trace import Trace\n",
+    "from trace_analysis import TraceAnalysis\n",
+    "\n",
+    "# Suport for FTrace events parsing and visualization\n",
+    "import trappy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test Environment set up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In case more than one Android device are conencted to the host, you must specify the ID of the device you want to target in `my_target_conf`. Run `adb devices` on your host to get the ID."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Setup a target configuration\n",
+    "my_target_conf = {\n",
+    "    \n",
+    "    # Target platform and board\n",
+    "    \"platform\"    : 'android',\n",
+    "    \n",
+    "    # Device ID\n",
+    "    \"device\" : \"00b1346f0878ccb1\",\n",
+    "    \n",
+    "    # Define devlib modules to load\n",
+    "    \"modules\"     : [\n",
+    "        'cpufreq'       # enable CPUFreq support\n",
+    "    ],\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "my_tests_conf = {\n",
+    "\n",
+    "    # Folder where all the results will be collected\n",
+    "    \"results_dir\" : \"AndroidN5X_Youtube\",\n",
+    "\n",
+    "    # Platform configurations to test\n",
+    "    \"confs\" : [\n",
+    "        {\n",
+    "            \"tag\"            : \"youtube\",\n",
+    "            \"flags\"          : \"ftrace\",           # Enable FTrace events\n",
+    "            \"sched_features\" : \"ENERGY_AWARE\",     # enable EAS\n",
+    "        },\n",
+    "    ],\n",
+    "    \n",
+    "    # FTrace events to collect for all the tests configuration which have\n",
+    "    # the \"ftrace\" flag enabled\n",
+    "    \"ftrace\"  : {\n",
+    "         \"events\" : [\n",
+    "            \"sched_switch\",\n",
+    "            \"sched_load_avg_cpu\",\n",
+    "            \"cpu_frequency\",\n",
+    "            \"cpu_capacity\"\n",
+    "         ],\n",
+    "         \"buffsize\" : 10 * 1024,\n",
+    "    },\n",
+    "    \n",
+    "    # Tools required by the experiments\n",
+    "    \"tools\"   : [ 'trace-cmd' ],\n",
+    "}"
+   ]
+  },
+  {
```

Comment:
I would like to run this notebook using SysTrace as well.
Thus it could be useful to add a cell right here where we ensure to have ADB running with root permissions:

```python
# Ensure ADB has root priviledges, which are required by systrace
!adb root
```